### PR TITLE
docs(match3): add meta merge match3 game tutorial

### PR DESCRIPTION
**Pull request by Kay0m**
# Pull request checklist

Use the following template to make sure your PR fits the ConsenSys documentation standard.

## Before creating the PR

Make sure that:

- [ ] You've read the [contribution guidelines](https://docs-template.consensys.net/getting-started/style-guide).
- [ ] You've previewed your changes locally.

## Impacted parts

For content changes:

- [ ] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've previewed your changes on Vercel below.

### CODE REVIEW

File: docs/use-linea/explore/use-match3-game.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+![](../../assets/meta_merge/match3_intro.png)
+
+I hope this information helps! If you have any other questions, feel free to ask me.
+
+## To Start:
+
+### Visit Meta Merge Match3 and Login.
+
+Visit the Meta Merge official website: https://match3-linea.metamerge.xyz/ and youwill find that there are 2 ways of logging in.
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-match3-game.md:31]: this is a broken link

File: docs/use-linea/explore/use-match3-game.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+![](../../assets/meta_merge/match3_select_wallet.png)
+
+After players click on the cards in the central area of the game, the cards will move up to the top frame. The top frame can store a total of 7 cards. When there are 3 identical cards, they will be eliminated, for example, three of the same cards will result in an elimination. Then click "Play" and let's get started!
+
+![](../../assets/meta_merge/match3_game_screenshot.png)
+
+When there are no cards that can be eliminated by forming a set of 3 on the game board, players can observe the temporarily unused cards in the lower layer. By clicking on the cards in the upper layer, the cards in the lower layer become available for use. However, when using this technique, it is important to avoid accumulating useless cards in hand, as it can ultimately lead to game failure.
+
+If the cards in hand has already been 7 and there are not 3 indentical cards here. Then unfortunately
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-match3-game.md:50]: Screenshot 2023-06-13 at 10 19 54 AM
Can you fix all the spelling errors?

If it needs to be a real word, you can:
- allowlist on `project-words.txt`
- run `npm run lint` to check if it's correct
- fix, squash commits, and force push

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]

File: docs/use-linea/explore/use-match3-game.md (hunk 1 above)
Comment (Kay0m) [reply docs/use-linea/explore/use-match3-game.md]: thanks, it is accessible now.

File: docs/use-linea/explore/use-match3-game.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Meta Merge Match3 Linea Game
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-match3-game.md:1]: Can you add the header

---
title: Use MetaMerge
sidebar_position: 1
---

Comment (emilyJLin95) [inline docs/use-linea/explore/use-match3-game.md:1]:
```suggestion
---
title: Use MetaMerge
sidebar_position: 1
---

# Meta Merge Match3 Linea Game
```

**Pull request merged**